### DISCUSSION TITLE
Simultaneous Sink and Source to a Bluetooth Headset?

### DISCUSSION THREAD

**Discussion by Ethan-Poag** [pschatzmann/ESP32-A2DP#353] [Q&A] [answered]
I am working on a project where I need a device that can connect to a Bluetooth headset, then simultaneously send and receive audio to and from the headset. Can I use these 2 libraries at the same time to send and receive audio from a BT headset? Thanks.

**Comment by pschatzmann** [comment, marked as answer]
No, see https://github.com/pschatzmann/ESP32-A2DP/discussions/283
